docker-compose.yml

```diff
@@ -15,6 +15,11 @@ services:
         environment:
             API_URI: http://web:1080
             APP_SECRET:
+    pattern_library:
```

Comment:
The `dummy-api_fpm` convention uses the name of the original project plus a suffix if there is a container image type to convey; this would result in `pattern-library`